packages/readme.md

```diff
@@ -0,0 +1,233 @@
+# Structure of package.mk files
+
+## Introduction
+
+The package.mk file defines variables and functions to build a package.
+
+## Variables
+All avialible variable, to control the build behavoir of your package.
+Please use these in the order, listed here.
+
+#### Base
+
+| Variable    | Default | Required |Description |
+|-------------|---------|----------|------------|
+| PKG_NAME    | -       | yes | Name of the packaged software application. Should be lowercase |
+| PKG_VERSION | -       | yes | Version of the packaged software application |
+| PKG_SHA256  | -       | yes | SHA256 hashsum of the application download file |
+| PKG_ARCH    | any     | no  | Architectures for which the package builds. `any` or a space separated list of `aarch64`, `arm` or `x86_64` |
+| PKG_LICENSE | -       | yes | License of the software application. [Reference](https://github.com/LibreELEC/LibreELEC.tv/tree/master/licenses) |
+| PKG_SITE    | -       | yes | Site of the software application |
+| PKG_URL     | -       | yes | Address at which the source of the software application can be retrieved |
+| PKG_MAINTAINER | -    | no  | Your name |
+| PKG_DEPENDS_BOOTSTRAP<br>PKG_DEPENDS_HOST   PKG_DEPENDS_INIT   PKG_DEPENDS_TARGET | - | no | A space separated list of name of packages required to build the software application |
+| PKG_SECTION | -       | no  | virtual if the package only defines dependencies |
+| PKG_SHORTDESC | -     | no<br>yes&nbsp;for&nbsp;addons | Short description of the application software used in various parts of Kodi |
+| PKG_LONGDESC | -      | yes | Long description of the application software used in various parts of Kodi |
+
+#### Universal Build Option
+| Variable    | Default | Required |Description |
+|-------------|---------|----------|------------|
+| PKG_SOURCE_DIR | -    | no  | Name of the folder to which the source of the software application unpacks. Should only be specified if the source of the software application unpacks to a folder whose name does not start with `$PKG_NAME` |
+| PKG_SOURCE_NAME | -   | no  | Name of the file of the source of the software application. Should only be specified if the source of the software application is not the basename of PKG_URL |
+| PKG_PATCH_DIRS | -    | no  | Change the path for pachtes, to apply to source. Normally the path `./patches` is used. Is this variable is set, the normal path will extended with the value of this variable as a subdirectory: `./patches/${PKG_PATCH_DIRS}` |
+| PKG_NEED_UNPACK | -   | no  | ??? can anyone explane, i have no idea. |
+| PKG_TOOLCHAIN | auto  | no  | Control which of the build toolchains is used. For detailed information, see the [Reference](#toolchain-options). |
+| PKG_BUILD_FLAGS | -   | no  | A Space seperated list of flags, which control often used build modification. Flags can be enabled or disables with a prefixed `+`/`-`. For detailed information, see the [Reference](#build_flags-options). |
```

Comment:
s/seperated/separated (also repeated in line 117)
", which control often used build modification." - maybe instead: "with which to fine-tune the build process."?
s/disables/disabled - "Flags can be enabled or disabled with a `+` or `-` prefix." may also read a little more easily.